lib/object.js

```diff
@@ -179,16 +179,16 @@ var out = {
      */
     forEach: function(cb) {
         var filename, relativePath, file;
+        /* jshint ignore:start */
+        // ignore warning about unwanted properties because this.files is a null prototype object
         for (filename in this.files) {
```

Comment:
@Stuk we could use `for(filename of Object.keys(this.files)) {` here instead, but it seems `jszip` is targetting es3 in `jshintrc`.

We could set this to `es6` but I assume this would break for a number of users and you would want to avoid that?

This is the error that comes up if I change it:

```
   ./lib/object.js
    182 |        for (filename of Object.keys(this.files)) {
                               ^ 'for of' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
```